test/spec/ApiConfigurationSpec.ts

```diff
@@ -0,0 +1,280 @@
+import { DefaultApiConfigurationParser } from "configuration/apiConfigurationParser";
+import { SubscriberEventType } from "configuration/apiConfiguration";
+
+const xml = `<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If the intention is testing the parser I would keep it here, but instead of going for a full valid document, I would go for a minimalist one, just to test the parser, and see if the code retrieves the necessary information from it.